android/src/main/java/com/adyenreactnativesdk/component/googlepay/AdyenGooglePayComponent.kt

```diff
@@ -23,6 +27,7 @@ import org.json.JSONException
 class AdyenGooglePayComponent(context: ReactApplicationContext?) : BaseModule(context) {
 
     private var googlePayComponent: GooglePayComponent? = null
+    private var loadingDialogFragment: PendingPaymentDialogFragment? = null
```

Comment:
good to name the field `pendingPaymentDialogFragment`

Reply:
Fixed here [0b73840](https://github.com/Adyen/adyen-react-native/pull/57/commits/0b738402faef32a6da86e6891f0fafed4675db56)